pkg/config/editor_presets.go

```diff
@@ -37,11 +37,12 @@ type editPreset struct {
 
 func getPreset(osConfig *OSConfig, guessDefaultEditor func() string) *editPreset {
 	presets := map[string]*editPreset{
-		"vi":    standardTerminalEditorPreset("vi"),
-		"vim":   standardTerminalEditorPreset("vim"),
-		"nvim":  standardTerminalEditorPreset("nvim"),
-		"emacs": standardTerminalEditorPreset("emacs"),
-		"nano":  standardTerminalEditorPreset("nano"),
+		"vi":      standardTerminalEditorPreset("vi"),
+		"vim":     standardTerminalEditorPreset("vim"),
+		"nvim":    standardTerminalEditorPreset("nvim"),
+		"emacs":   standardTerminalEditorPreset("emacs"),
+		"nano":    standardTerminalEditorPreset("nano"),
+		"kakoune": standardTerminalEditorPreset("kakoune"),
```

Comment:
I think there is an issue with this line. The command for kakoune is `kak` not `kakoune`. So it should say:
```
"kakoune": standardTerminalEditorPreset("kak"),
```

Reply:
Oops, you are right, sorry for missing this. Do you feel like making a trivial PR for this?

Reply:
Yeah I can do that

Reply:
https://github.com/jesseduffield/lazygit/pull/2708/files